bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -2003,19 +1979,6 @@
             UPDATE concept_set, concept_name SET concept_set.sort_weight = 9 where concept_set.concept_id = concept_name.concept_id and concept_name.name = 'Unit(s)' and concept_set.concept_set = @dosing_units_concept_id;
         </sql>
     </changeSet>
-    <changeSet id="bahmni-core-201409301255" author="Chethan,Banka">
```

Comment:
Any specific reason for removing this changeset? are we importing this from elsewhere?

Reply:
This set mapping happens for a concept named Dose Quantity Units which is not created from any migration/script and there is no other reference to this concept as well in code.(In lite environments we dont have this concept at all) Concept only exists in older DB backups, so may be manually created.